cli/internal/runsummary/format_execution_summary.go

```diff
@@ -26,14 +32,37 @@ func (rsm *Meta) printExecutionSummary() {
 		}
 	}
 
-	if summary.ExecutionSummary.attempted == 0 {
+	if attempted == 0 {
 		ui.Output("") // Clear the line
 		ui.Warn("No tasks were executed as part of this run.")
 	}
 
-	ui.Output("") // Clear the line
-	ui.Output(util.Sprintf("${BOLD} Tasks:${BOLD_GREEN}    %v successful${RESET}${GRAY}, %v total${RESET}", summary.ExecutionSummary.cached+summary.ExecutionSummary.success, summary.ExecutionSummary.attempted))
-	ui.Output(util.Sprintf("${BOLD}Cached:    %v cached${RESET}${GRAY}, %v total${RESET}", summary.ExecutionSummary.cached, summary.ExecutionSummary.attempted))
-	ui.Output(util.Sprintf("${BOLD}  Time:    %v${RESET} %v${RESET}", time.Since(summary.ExecutionSummary.startedAt).Truncate(time.Millisecond), maybeFullTurbo))
+	ui.Output("")    // Clear the line
+	spacer := "    " // 4 chars
+
+	// We'll start with some default lines
+	lines := []string{
+		util.Sprintf("${BOLD} Tasks:${BOLD_GREEN}%s%v successful${RESET}${GRAY}, %v total${RESET}", spacer, success, attempted),
+		util.Sprintf("${BOLD}Cached:%s%v cached${RESET}${GRAY}, %v total${RESET}", spacer, cached, attempted),
+		util.Sprintf("${BOLD}  Time:%s%v${RESET} %v${RESET}", spacer, duration, maybeFullTurbo),
+	}
+
+	// If we have a run summary file and we can get  a
+	if rsm.getPath().FileExists() {
+		if relativePath, err := rsm.repoRoot.PathTo(rsm.getPath()); err == nil {
```

Comment:
One thing to figure out is if a relative path from the repoRoot is confusing if we're running from a subdir 🤔